lib/virt_autotest/common.pm

```diff
@@ -253,23 +253,23 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             name => 'sles16efi_online',
             extra_params => '--os-variant sles16',    # Use SLES16 variant
             distro => 'SLE_16',
-            iso_url => 'http://openqa.suse.de/assets/iso/SLES-16.0-Online-x86_64-Build135.5.install.iso',    # ISO download URL
+            iso_url => 'http://openqa.suse.de/assets/iso/SLES-16.0-Online-x86_64-GM.install.iso',    # ISO download URL
             boot_firmware => 'efi',    # SLES16 only supports EFI guests, no BIOS support
         },
         sles16efi_full => {
             name => 'sles16efi_full',
             extra_params => '--os-variant sles16',    # Use SLES16 variant
             distro => 'SLE_16',
-            location => 'http://openqa.suse.de/assets/repo/SLES-16.0-Full-x86_64-Build135.5.install/',    # Network location for live OS
-            install_url => 'http://openqa.suse.de/assets/repo/SLES-16.0-x86_64-Build135.5/',    # Install repository URL
+            location => 'https://openqa.oqa.prg2.suse.org/assets/repo/fixed/SLES-16.0-Full-x86_64-GM.install/',    # Network location for live OS
+            install_url => 'https://openqa.suse.de/assets/repo/fixed/SLES-16.0-x86_64-GM/',    # Install repository URL
```

Comment:
For full, we can use https://openqa.suse.de/assets/repo/fixed/SLES-16.0-Full-x86_64-GM.install/ as repo. I think it is not needed to use full iso for installation, right?

Reply:
It is just a test.
I will submit another PR to separate the issues.